engine/crates/integration-tests/tests/federation/auth/authenticated.rs

```diff
@@ -101,22 +101,22 @@ fn not_authenticated() {
 
         // We shouldn't have requested the field.
         let requests = engine.drain_graphql_requests_sent_to::<SecureSchema>();
-        insta::assert_json_snapshot!(requests, @r###"
+        insta::assert_json_snapshot!(requests, @r#"
         [
           {
-            "query": "query {\n  check {\n    anonymous\n  }\n}\n",
+            "query": "query{ check{ anonymous}}",
```

Comment:
Do we need the whitespace after `{`'s?

Reply:
It's just to handle correctly multiple fields. one space won't matter, the actual problematic part were newlines